src/pages/feeds/UpdateFeed.tsx

```diff
@@ -76,10 +73,10 @@ export default function UpdateFeed(props: RouteComponentProps<MatchParams>): Rea
     }
 
     try {
-      await updateFeed(beeApi, identity, props.match.params.hash, selectedStamp, password as string)
+      await updateFeed(beeApi, identity, hash!, selectedStamp, password as string)
```

Comment:
I don't really know under what circumstances the hash would be undefined. Anyways `updateFeed` function does validation AFAIK.